src/Microsoft.Health.Fhir.Liquid.Converter.UnitTests/Models/ComplexObjectFilterUtilityTestCase.cs

```diff
@@ -0,0 +1,23 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.Fhir.Liquid.Converter.UnitTests.Models
+{
+    public struct ComplexObjectFilterUtilityTestCase
+    {
+        public object[] Input;
+        public string Path;
+        public string Value;
+        public object[] Expected;
+
+        public ComplexObjectFilterUtilityTestCase(object[] input, string path, string value, object[] expected)
```

Comment:
Since the class members are public without readonly, you technically don't need a constructor. You could create an instance like this

```csharp
var x = new ComplexObjectFilterUtilityTestCase
{
	Input = new [] { new object() },
	Path = "Path",
	Value = "Value",
	Expected = new [] { new object() },
};
```

Reply:
Good to know but I'm actually not a fan of writing the names so much.